contracts/cw20-merkle-airdrop/src/msg.rs

```diff
@@ -0,0 +1,64 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use cosmwasm_std::Uint128;
+
+#[derive(Serialize, Deserialize, JsonSchema)]
+pub struct InstantiateMsg {
+    pub owner: String,
+    pub cw20_token_address: String,
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum ExecuteMsg {
+    UpdateConfig {
+        owner: Option<String>,
+    },
+    RegisterMerkleRoot {
+        merkle_root: String,
```

Comment:
Please add rust doc that this is "hex-encoded merkle root" 
(and same with Claim.proof)